Test/HUnit/Base.hs

```diff
@@ -2,6 +2,7 @@
 #if MIN_VERSION_base(4,8,1)
 #define HAS_SOURCE_LOCATIONS
 {-# LANGUAGE ImplicitParams #-}
+{-# LANGUAGE ConstrainedClassMethods #-}
```

Comment:
BTW, I guess only this change is required to make HUnit build with GHC 8.0, right?  In case you need this on Hackage ASAP, let's have this as a separate commit, and I'll already release it separately.